### PR TITLE
Updated nix user documentation to reflect Flakes deprecation

### PR DESCRIPTION
**Pull request by andriy-safe-ai**

### CODE REVIEW

**Comment by steven-basart** [comment]
Fix title so it can fit on one line

File: _posts/2023-05-03-cais-cluster-documentation.md [hunk 1]
```diff
@@ -126,84 +126,85 @@ By default, all users of the cluster are limited to 1 TB of file system storage
 # Package Management
 ## Nix Package Manager
 
-Nix is a package manager for Linux and other Unix systems, with a wide selection of up-to-date packages. This document is a brief introduction to Nix and how to use it. 
+Nix is a package manager for Linux, MacOS, Windows, and other Unix systems, with a wide selection of up-to-date packages. This document is a brief introduction to Nix and how to use it.
```
Comment (steven-basart) [inline _posts/2023-05-03-cais-cluster-documentation.md:129]: any modern operating system,

File: _posts/2023-05-03-cais-cluster-documentation.md [hunk 2]
```diff
@@ -126,84 +126,85 @@ By default, all users of the cluster are limited to 1 TB of file system storage
...
-
-# Install a package from a specific branch:
-nix profile install nixpkgs/release-20.09#hello
+nix-install hello neofetch
 ```
 
 ### How to list installed packages
 
-Use `nix-list`, an alias for `nix profile list`, to list installed pacakges.
+Use `nix-list`, an alias for `nix-env -q`, to list installed pacakges.
```
Comment (steven-basart) [inline _posts/2023-05-03-cais-cluster-documentation.md:157]: typo it's pak-ages

File: _posts/2023-05-03-cais-cluster-documentation.md (hunk 1 above)
Comment (steven-basart) [reply _posts/2023-05-03-cais-cluster-documentation.md:129]: We should add a statement that support through Nix for us is experimental or something along those lines.  The fact that we're wiping out all of the users packages means it isn't stable and might even be deprecated in the future for something else.

**Review by steven-basart — CHANGES_REQUESTED** [review]
Minor changes

**Review by steven-basart — APPROVED** [review]
LGTM

**Comment by andriy-safe-ai** [comment]
Rebased to one commit. Merged.

**Pull request merged**